src/modules/colorPicker/ColorPickerUI/Behaviors/MoveWindowBehavior.cs

```diff
@@ -7,21 +11,19 @@ namespace ColorPicker.Behaviors
 {
     public class MoveWindowBehavior : Behavior<Window>
     {
-        public static DependencyProperty LeftProperty = DependencyProperty.Register("Left", typeof(double), typeof(MoveWindowBehavior), new PropertyMetadata(new PropertyChangedCallback(LeftPropertyChanged)));
+        private static readonly DependencyProperty LeftProperty = DependencyProperty.Register("Left", typeof(double), typeof(MoveWindowBehavior), new PropertyMetadata(new PropertyChangedCallback(LeftPropertyChanged)));
```

Comment:
Please revert this, all DependencyProperties should be public (unless using some very specific scenarios, which are not present here)

> It is not generally useful, and it is potentially misleading, to declare a DependencyProperty as any access level other than public. 

[Dependency Property Security](https://docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/dependency-property-security#:~:text=Dependency%20properties%20should%20generally%20be%20considered%20to%20be,make%20security%20guarantees%20about%20a%20dependency%20property%20value.)

> By virtue of not being settable, read-only dependency properties aren't appropriate for many of the scenarios for which dependency properties normally offer a solution (namely: data binding, directly stylable to a value, validation, animation, inheritance).

[Read-Only Dependency Properties](https://docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/read-only-dependency-properties)